pkg/export/setup/setup.go

```diff
@@ -117,7 +123,7 @@ func FromFlags(a *kingpin.Application, userAgent string) func(log.Logger, promet
 		Default(opts.ProjectID).StringVar(&opts.ProjectID)
 
```

Comment:
Suggest 
- moving `formatUserAgentSuffix` into this file
- creating a new user-agent suffix variable
- calling `formatUserAgentSuffix` before passing it into ops.UserAgentSuffix.
- removing the need of opts.Environment

So we are passing in true userAgentSuffix to be used for `newMetricClient` in export.go.

For example:

```
userAgentSuffx := formatUserAgentSuffix(userAgent, env)
```

Let me know what you think.